esrally/driver/runner.py

```diff
@@ -2042,15 +2124,26 @@ async def __call__(self, es, params):
         success = True
         searches = mandatory(params, "retrieve-results-for", self)
         request_params = params.get("request-params", {})
-        for search_id, _ in async_search_ids(searches):
+        stats = {}
+        for search_id, search in async_search_ids(searches):
             response = await es.async_search.get(id=search_id,
                                                  params=request_params)
-            success = success and not response["is_running"]
+            is_running = response["is_running"]
+            success = success and not is_running
+            if not is_running:
+                stats[search] = {
+                    "hits": response["response"]["hits"]["total"]["value"],
```

Comment:
Do we document these as the "meta-fields" available for assertions?

Reply:
As indicated above: No, it's currently undocumented.